xds/internal/xdsclient/transport/transport.go

```diff
@@ -429,7 +439,7 @@ func (t *Transport) recv(stream adsStream) bool {
 		resources, url, rVersion, nonce, err := t.recvAggregatedDiscoveryServiceResponse(stream)
 		if err != nil {
 			t.adsStreamErrHandler(err)
-			t.logger.Warningf("ADS stream is closed with error: %v", err)
+			t.logger.Warningf("ADS stream closed: %v", err)
```

Comment:
nit: Shouldn't the log message indicate that it was `Recv()` on the stream that failed?

Reply:
It doesn't really matter if `Recv()` or `Send()` failed on the stream. In fact in many cases, `Send()` doesn't return a useful error and we rely on the error from `Recv()`. All that matters here is a log statement saying the stream has failed, and the error should indicate why.

Reply:
I think my question is more if ADS stream closing the only condition where recv() would fail?